.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: python
 python: 3.6
 cache: pip
+
 before_install:
-- pip install -U pre-commit
-- pip install pylint-django==0.11.1
-- pip install Django==2.0.5
-script: pre-commit run --all-files
+- pip install -r requirements/test.txt
+
+script:
+- pre-commit run --all-files
```

Comment:
Don't modify lines you're not actually changing.

Reply:
Sorry @webknjaz I think I do not understand what I need to do 

> Please use a sequence instead of a string, it makes diffs nicer.

Reply:
yes, but for lines you actually change, not for random unrelated lines.